config.dev.toml

```diff
@@ -2,6 +2,9 @@
 # Most properties can be overridden with environment variables specified in this file
 # Most properties also have defaults (mentioned in this file) if they are not set in either the config file or the corresponding environment variable
 
+# If you put this file where you start Athens from (or in your home directory) as ".athens.toml", athens will use it when starting.
```

Comment:
this says `.athens.toml` but the `defaultConfigFile` says `athens.toml`. I think I like `.athens.toml` better but either way is cool with me 👍 

It also mentions the home directory but I see you removed it from the code. Also either way is cool with me 😄